src/client/gui/lib/vm_details/ip_addresses.dart

```diff
@@ -1,12 +1,14 @@
 import 'package:flutter/material.dart' hide Tooltip;
+import '../copyable_text.dart';
```

Comment:
please put this import in the same group as the other imports that come from our own code, in alphabetical order. sucks that `dart format` does not tell us about this :/